src/dexpo.cpp

```diff
@@ -77,27 +76,37 @@ main ()
         case XCB_EXPOSE:
           {
             w.draw_gui ();
-            // w.draw_border (0, dexpo_hlcolor);
+            for (int i = 0; int (i < w.desktops.size ()); i++)
+              {
+                w.draw_border (i, dexpo_desktop_color);
+              }
+            w.desktop_sel = get_property_value (
+                window::c_, window::screen_->root, "_NET_CURRENT_DESKTOP")[0];
```

Comment:
bro there is a function `get_current_desktop` in `xcb_utils.cpp`

Reply:
fuck